src/main/groovy/org/mockito/release/notes/internal/DefaultImprovement.java

```diff
@@ -87,4 +87,40 @@ public String toJson() {
     public void toJson(Writer writable) throws IOException {
         writable.append(toJson());
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
equals/hashcode methods are relevant for tests. Without it the current assertions in `DefaultContributionSetSerializerTest` will not work. 

The previous assertions:

```
EqualsBuilder.reflectionEquals(defaultContributionSet.getAllCommits(), deserializedData.getAllCommits())
```

also didn't work, I don't know why.